03/app.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener('DOMContentLoaded', function(){
+
+    const btns = document.querySelectorAll('button');
+    const clickEvent = function(el){
+
+        
+        this.innerText = 'clicked';
+        
+        if(this.innerText === 'clicked') {
+            el.stopImmediatePropagation();
+            console.log(this.innerText);
+        }
+
+    }
+
+    if(btns) {
```

Comment:
Tutaj nie trzeba, bo mamy wiele elementów wyszukiwanych. Jeśli nie będzie żadnego to będzie pusta tablica, pętla się nie wykona ale nie będzie błędu ;)